src/index.js

```diff
@@ -1,12 +1,14 @@
 import 'babel-polyfill';
 import 'source-map-support/register';
 
-import debug from 'debug';
+import debugLog from 'debug';
 import fs from 'graceful-fs';
 import mkdirp from 'mkdirp';
```

Comment:
the method 'ensurePath' uses mkdirp and is being used across all projects. I would prefer to add it as a dependency to the package.json.